packages/@o3r/rules-engine/schemas/rulesets.schema.json

```diff
@@ -51,23 +51,16 @@
           "additionalItems": false
         },
         "linkedComponent": {
-          "type": "object",
+          "$ref": "#/definitions/LinkedComponent",
           "description": "Component linked to the ruleset, if set it will disable the ruleset execution per default, waiting to a subscription",
-          "properties": {
-            "library": {
-              "type": "string",
-              "description": "Name of the component library"
-            },
-            "name": {
-              "type": "string",
-              "description": "Name of the component"
-            }
-          },
-          "additionalItems": false,
-          "required": [
-            "library",
-            "name"
-          ]
+          "deprecated": true
+        },
+        "linkedComponents": {
+          "type": "array",
+          "description": "Components linked to the ruleset. If at least one present, it will disable the ruleset execution by default, waiting to a subscription",
```

Comment:
```suggestion
          "description": "Components linked to the ruleset. If present, it will disable the ruleset execution by default, waiting for at least one subscription",
```